consensus/istanbul/core/handler.go

```diff
@@ -121,11 +121,17 @@ func (c *core) handleEvents() {
 					//c.backend.Gossip(c.valSet, p)
 				}
 			}
-		case _, ok := <-c.timeoutSub.Chan():
-			if !ok {
+		case ev, ok := <-c.timeoutSub.Chan():
+			if !ok || ev.Data == nil {
+				logger.Warn("Invalid message from timeout channel")
```

Comment:
Is receiving a `nil` message possible? If not how about leaving the log as `Error`?

Reply:
Applied, thanks.